app/streaming/trace/service/service_test.go

```diff
@@ -0,0 +1,73 @@
+package service
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	codecTypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/tendermint/tendermint/libs/log"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+	types1 "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+var (
```

Comment:
nit: Global variables are an easy way to end up with broken things.

I admit that It's probably safe in this limited instance, but is something to avoid in most cases (in favor of the `"github.com/stretchr/testify/suite"` approach).